ipaserver/install/server/replicainstall.py

```diff
@@ -1049,6 +1058,14 @@ def promote_check(installer):
                 logger.error("Certificates could not be provided when "
                              "CA is present on some master.")
                 raise ScriptError(rval=3)
+            if options.setup_ca and options.server and \
+               ca_host != options.server:
+                # Installer was provided with a specific master
+                # but this one doesn't provide CA
+                logger.error("The specified --server %s does not provide CA, "
+                             "please provide a server with the CA role",
```

Comment:
Hello @flo-renaud 
Can we use some file to place the stderr/stdout messages? something like `ipaplatform.paths`, so that testsuite can import this file and use the stderr/stdout from it.
I have seen failures in test due to change in stdout/stderr messages(downstream observation though).

Reply:
Hi @mrizwan93 
The error messages are not something we commit to (as opposed to the exit codes for instance, or the APIs), and I am reluctant to define constants for them. This would lead to a nightmare finding a different constant name for each message.
If you prefer having a test that doesn't rely on the expected message, I would suggest checking the exit code instead (the value 4 will be documented for this specific case).

Reply:
Okay, relying on returncode make sense.